nucliadb_node/nucliadb_node/tests/fixtures.py

```diff
@@ -214,16 +229,17 @@ def data_path():
 
 
 @pytest.fixture(scope="function")
-def shadow_folder(data_path):
+def shadow_folder(data_path) -> Iterable[str]:
```

Comment:
NIT: typing would be simpler if we simply return the string

Reply:
Yes, but as it's an async generator, mypy complains if you annotate only with `str`

Reply:
What I meant is that you can change the `yield` to a return and then the annotation just a `str`